modules/processing.py

```diff
@@ -65,8 +65,12 @@ def apply_color_correction(correction, original_image):
 def uncrop(image, dest_size, paste_loc):
     x, y, w, h = paste_loc
     base_image = Image.new('RGBA', dest_size)
+    factor_x = w // image.size[0]

```

Comment:
as mentioned in the other PR I believe this is incorrect
- https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/16243#discussion_r1817835780